dev/codesniffer/ruleset.xml

```diff
@@ -3,11 +3,16 @@
 <ruleset name="Dolibarr">
 	<description>Dolibarr coding standard.</description>
 
-	<exclude-pattern>*/conf.php</exclude-pattern>
-	<exclude-pattern>*/includes/*</exclude-pattern>
-	<exclude-pattern>*/documents/*</exclude-pattern>
-	<exclude-pattern>*/dev/vagrant/*</exclude-pattern>
-	<exclude-pattern>*/custom/*</exclude-pattern>
+	<exclude-pattern>build/html</exclude-pattern>
+	<!--
```

Comment:
Why did not you keep pattern  */documents/*  so it is not bugged. Only dir is excluded